### ISSUE TITLE
Fix validation loss error in VAT segmentation model 

### ISSUE THREAD

**Issue by gautham-phenomx** [gautham-phenomx/SAT-VAT#5]
Train Loss - 1.2
Val loss - 12.1
Fix the error difference
